hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -497,4 +489,32 @@ private void flushToDiskIfRequired(HoodieRecord record) {
       numberOfRecords = 0;
     }
   }
+
+  protected void appendDataAndDeleteBlocks(Map<HeaderMetadataType, String> header) {
```

Comment:
@alexeykudinkin can you call out what has changed in this block of code?

Reply:
Frankly, can't understand why this method got moved. Nothing should have changed here. Let me revert this

Reply:
Oh, i've found the root-cause of those inexplicable changes -- i guess after upgrade of IDEA i've got new box "Rearrange code" ticked automatically (never ticked it myself) and i've confirmed it being the root-cause of those random re-orderings

<img width="357" alt="Screen Shot 2022-01-13 at 6 10 49 PM" src="https://user-images.githubusercontent.com/428277/149439483-0d67b659-a61c-4ffe-807e-70a07e9d7d0e.png">